packages/common/constants.js

```diff
@@ -233,29 +233,42 @@ export const PERMISSION_ENTITY = {
 export const PERMISSION_READ = 'read';
 export const PERMISSION_WRITE = 'write';
 
+export const ROLE_ADMIN = 'ROLE_ADMIN';
```

Comment:
Pls, remove the export property and separate the roles.
Somebody can, by chance, use a role directly instead of using a business logic role.
Here are the different roles: TEAM (ROLE_ADMIN, ROLE_MANAGER, ROLE_MEMBER) and DOMAIN (ROLE_ADMIN, ROLE_USER, ROLE_READ_ONLY_USER, ROLE_ANONYMOUS_USER ); these roles should be under their business logic role.